ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                           </msItem>
+                           <msItem xml:id="ms_i1.7">
+                             <locus from="185vb" to="191vb"/>
+                             <title type="complete" xml:lang="gez">Further prayers and readings</title>
+                             <textLang mainLang="gez"/>
+                               <incipit xml:lang="gez"><hi rend="rubric">ካህን፡ ይብል፡ ሰብሐት፡ ለእግዚአብሔር፡ ወሕዝብ፡ ይብሉ፡ ርትዕ᎓᎓</hi> ለከ᎓
+                               እ<pb n="186ra"/>ግዚኦ᎓ ለገባሬ፡ ኵሉ፡ <sic>ዘኢያስቲርኢ፡</sic> ንሰፍሕ፡ ነፍስነ፡ <sic resp="DN">ወሰብሐተ፡</sic>፡ ዘነግህ፡
+                             ንፈኑ፡ ለከ፡  </incipit>
+                             </msItem>
+                               <msItem xml:id="ms_i1.8">
+                                 <locus from="191rb" to="191vb"/>
+                                 <title type="complete" xml:lang="gez" ref="LIT4262Prayer"><hi rend="rubric">ጸሎተ፡ ሃይማኖት፡</hi></title>
+                                 <textLang mainLang="gez"></textLang>
+                                   <incipit xml:lang="gez">ነአምን፡ በአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ እምቅ<pb n="191va"/>ድመ፡ ዓለም፡ ወመዋዕል፡ እምቅድመ፡ ጊዜ፡ ወሰዓት፡ ፡
+                                     ክርስቲያን፡
+                                 </incipit>
+                               </msItem>
+                           <msItem xml:id="ms_i1.9">
+                             <locus from="191vb" to="194ra"/>
+                             <title type="complete" xml:lang="gez" ref="LIT6291HassabaRetu">A treatise on computus</title>
+                             <textLang mainLang="gez"></textLang>
+                               <incipit xml:lang="gez"><hi rend="rubric">በስመ᎓ አብ᎓ ወወልድ᎓ ወመን</hi>ፈስ፡ ቅዱስ፡
+                               አሐዱ፡ አምላክ፡ <hi rend="rubric">ጸሐፍና፡ በዘንዜክር፡ ሐሳ</hi>በ፡ ርቱዕ፡
+                             </incipit>
+                             </msItem>
+                      </msItem>
+                           <msItem xml:id="ms_i2">
+                             <locus from="194ra" to="194rb"/>
+                              <title type="complete" xml:lang="gez" ref="LIT3856KingList"/>
+                              <textLang mainLang="gez"/>
+                              <incipit xml:lang="gez">ዝውእቱ᎓ አስማቲሆሙ፡ ለነግሥተ፡ ኢትዮጵያ፡ ወለነገሥተ፡ አክሱም፡
+                              እምቅድመ፡ ልደተ፡ ክርስቶስ፡ ብኩሩ፡ ለሰሎሞን፡ ንጉሥ፡ እንበይነ፡ ላኅክም፡ ወልደ፡ ለማኪዳ፡ ንግሥተ፡ አክሱም፡ ወእምድኅረሁ፡
+                            ብእሲ፡ አንጋቦ፡</incipit>
+
+                             <colophon xml:id="coloph1" xml:lang="gez">
+                               <locus target="#199va"></locus>
+                          <note>Edited and translated in <bibl>
+                             <ptr target="bm:ContiRossini1914Notice"/>
+                             <citedRange unit="page">47</citedRange>
+                          </bibl>. The colophon mentions the date <date>51 year of grace</date>
+                          corresponding to <date>1475 EC</date>, the time of <persName ref="PRS2334BaedaM">King Baʾǝda Māryām I</persName>,
```

Comment:
```
The colophon mentions the date <date calendar="ethiopian" when="51">51 year of grace</date>
                          corresponding to <date when-custom="1475" notBefore="1482" notAfter="1483">1475 EC</date>
```

Reply:
If I encode like this, the system reports a mistake.

Reply:
I guess it wants 0051

Reply:
Indeed. In a couple of manuscripts there is an inconsistency in the use of "ethiopian" in relation to "years of mercy" (= Ethiopian Calendar, +7/8 years > common era) that is called "era of grace". But this is a different matter (the era that starts 283/284 CE + 76 years). Sometimes both ears get confused. I can correct them.